lib/src/test/java/io/ably/lib/test/realtime/RealtimeAuthTest.java

```diff
@@ -24,6 +24,16 @@
     @Rule
     public Timeout testTimeout = Timeout.seconds(30);
 
+    /**
+     * Verifies an Exception is thrown, when client is initialized with an empty key
+     *
+     * @throws IllegalArgumentException
+     */
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
Nice use of JUnit in the proper manner (a rare thing in this codebase, alas). ❤️